src/content/docs/iast/install.mdx

```diff
@@ -15,85 +15,115 @@ freshnessValidatedDate: never
 To install New Relic IAST:
 
 <Steps>
-  <Step>
-    Go to <DNT>**[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST**</DNT> and click <DNT>**Set up IAST with applications**</DNT>.
+ <Step>
+    Go to <DNT>**[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST**</DNT> and click <DNT>**Set up IAST with applications**</DNT> or <DNT>**Install**</DNT>.
 
     <img
-      title="Set up IAST with applications"
-      alt="Set up IAST with applications"
-      src="/images/iast_screenshot-full_install.webp"
-    />
-  </Step>
-
-  <Step>
-    Confirm IAST isn't running in a production environment and click <DNT>**Continue**</DNT>.  
-
-  </Step>
-
-  <Step>
-    Select the language of your application and complete the steps.
+        title="Set up IAST with applications"
+        alt="Set up IAST with applications"
+        src="/images/iast_screenshot-full_install.webp"
+    />    
+ </Step>
+ <Step>
+    Confirm that IAST isn't running in a production environment, and agree to the [service specific terms](https://newrelic.com/termsandconditions/service-specific),  and then click <DNT>**Continue**</DNT>.    
+ </Step>
+ <Step>
+    Select the  programming language of your application, update the respective APM agent, and click **Next**.
 
     <img
-      title="Install New Relic IAST"
-      alt="Install New Relic IAST"
-      src="/images/iast_screenshot-crop_install.webp"
-    />
-  </Step>
+        title="Install New Relic IAST"
+        alt="Install New Relic IAST"
+        src="/images/iast_screenshot-crop_install.webp"
+    />    
+ </Step>
+ <Step>
+    Based on the selected programming language, setup your security agent as follows and click **Next**.
```

Comment:
Instead of "programming language" I'd write something like "Based on your APM agent..." 

Also, I think it should be "set up", yeah?

Where is someone clicking Next there? These feel like two different steps. One is related to changing the APM config settings, then maybe something in the UI? I think that should go into another step